system/datatypes/mailinglist.php

```diff
@@ -660,116 +785,27 @@ private function createEditForm(array $data = []) {
 	 * @return WymHTMLForm
 	 */
 	private function createMailingForm(array $data = []) {
-		$idFieldName = self::FIELD_NAME_ID;
-		$subject = __('subject');
-		$subjectFieldName = self::FIELD_NAME_SUBJECT;
-		$message = __('message');
-		$messageFieldName = self::FIELD_NAME_MESSAGE;
 		$html = <<<EOF
-			<input type="hidden" id="$idFieldName" name="$idFieldName" />
+			<input type="hidden" id="[[field-name-id]]" name="[[field-name-id]]" />
 			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
-				<label for="$subjectFieldName"> $subject </label> <input type="text" id="$subjectFieldName" name="$subjectFieldName" />
+				<field-name-labelsubjectFieldName]]"> [[subject]] </field-name-label> <input type="text" id="[[subjectFieldName]]" name="[[subjectFieldName]]" />
```

Comment:
This looks broken: `<field-name-labelsubjectFieldName]]">`

Also, `</field-name-label>` is not valid HTML tag? What's going  on here?

Also, `[[subjectFieldName]]` should be `[[field-name-subject]]` I think?

Reply:
fixed it